test/node/common-forward-event.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const tap = require('tap');
+const sinon = require('sinon');
+
+const events = require('events');
+
+const common = require('../../lib/common');
+
+tap.test('must forward a single event', (test) => {
+  const sourceEventEmitter = new events.EventEmitter();
+  const targetEventEmitter = new events.EventEmitter();
+
+  const spy = sinon.spy();
+
+  common.forwardEvent(sourceEventEmitter, targetEventEmitter, 'testEvent');
+  targetEventEmitter.on('testEvent', spy);
+
+  sourceEventEmitter.emit('testEvent');
+  test.assert(spy.called, 'event handler must be called');
+  test.end();
+});
+
+tap.test('must forward a single event under a new name', (test) => {
+  const sourceEventEmitter = new events.EventEmitter();
+  const targetEventEmitter = new events.EventEmitter();
+
+  const spy = sinon.spy();
+
+  common.forwardEvent(sourceEventEmitter, targetEventEmitter,
+                        'testEvent', 'renamedEvent');
+  targetEventEmitter.on('renamedEvent', spy);
+
+  sourceEventEmitter.emit('testEvent');
+  test.assert(spy.called, 'event handler must be called');
```

Comment:
@lundibundi it should have been if it was a sentence, but in this case it's just a test name. Look at the other assertions, they are written in the same style.